### DISCUSSION TITLE
Can it set different scale factors for x/y/z axes directly?

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#814] [Q&A] [answered]
Hi all,
I want to show some points whose z coordinates are only 0.001 times of y coordinates.
I modify z coordinates by multiply 1000 so as to make them looks more spatial. But the z-axis labels are not their actual values.
I took a try but the scale function doesn't work. How to solve this problem?
Can it set different scale factors for x/y/z axes directly?

Thanks.

**Comment by marcomusy** [comment, marked as answer]
Ok ... creating different scales in a 3D world coordinate system is _not recommended_!
Yet, it's possible by resetting the axes scales manually with `z_values_and_labels` like in example [custom_axes1.py](https://github.com/marcomusy/vedo/tree/master/examples/pyplot/custom_axes1.py).

I would recommend setting the axis z-title text with something like `ztitle="my units *1000"`.

**Reply by ZiguoAtGitHub** [reply]
Thanks for your reply!
Maybe my case is a special one. The first solution you mentioned is better for my case.
